docs/source/API/core/view/resize.rst

```diff
@@ -0,0 +1,158 @@
+
+  * ``v``: existing view, can be a default constructed one.
+  * ``n[X]``: new length for extent X.
+  * ``arg_prop``: View constructor properties, e.g., ``Kokkos::view_alloc(Kokkos::WithoutInitializing)``.
+
+  Restrictions:
+
+  * ``View<T, P...>::array_layout`` is either ``LayoutLeft`` or ``LayoutRight``.
+  * ``arg_prop`` must not include a pointer to memory, a label, or a memory space.
+
+* .. code-block:: cpp
+
+     template <class T, class... P>
+     void resize(Kokkos::View<T, P...>& v,
+                 const typename Kokkos::View<T, P...>::array_layout& layout);
+
+  Resizes ``v`` to have the new dimensions while preserving the contents for the common subview of the old and new view.
+
+  * ``v``: existing view, can be a default constructed one.
+  * ``layout``: a layout instance containing the new dimensions.
+
+* .. code-block:: cpp
+
+     template <class T, class... P>
+     void resize(const I& arg_prop, Kokkos::View<T, P...>& v,
+	         const typename Kokkos::View<T, P...>::array_layout& layout);
+
+  Resizes ``v`` to have the new dimensions while preserving the contents for the common subview
+  of the old and new view. The new ``Kokkos::View`` is constructed using the View constructor
+  property ``arg_prop``, e.g., Kokkos::WithoutInitializing.
+
+  * ``v``: existing view, can be a default constructed one.
+  * ``layout``: a layout instance containing the new dimensions.
+  * ``arg_prop``: View constructor property, e.g., ``Kokkos::WithoutInitializing``.
+
+* .. code-block:: cpp
+
+     template <class T, class... P, class... ViewCtorArgs>
+     void resize(const Impl::ViewCtorProp<ViewCtorArgs...>& arg_prop,
+	         Kokkos::View<T, P...>& v,
+	         const typename Kokkos::View<T, P...>::array_layout& layout);
+
+  Resizes ``v`` to have the new dimensions while preserving the contents for the
+  common subview of the old and new view. The new ``Kokkos::View`` is constructed using
+  the View constructor properties ``arg_prop``, e.g., ``Kokkos::view_alloc(Kokkos::WithoutInitializing)``.
+  If ``arg_prop`` includes an execution space, it is used for allocating memory and for copying elements without using a final fence.
+
+  * ``v``: existing view, can be a default constructed one.
+  * ``layout``: a layout instance containing the new dimensions.
+  * ``arg_prop``: View constructor properties, e.g., ``Kokkos::view_alloc(Kokkos::WithoutInitializing)``.
+
+  Restrictions:
+
+  * ``arg_prop`` must not include a pointer to memory, a label, or a memory space.
+
+Example:
+--------
+
+* .. code-block:: cpp
+
+    Kokkos::resize(v, 2, 3);
+
+Resize a ``Kokkos::View`` with dynamic rank 2 to have dynamic extent 2 and 3 respectively preserving previous content.
+
+* .. code-block:: cpp
+
+    Kokkos::resize(Kokkos::WithoutInitializing, v, 2, 3);
+
+Resize a ``Kokkos::View`` with dynamic rank 2 to have dynamic extent 2 and 3 respectively preserving previous content. After this call, the new content is uninitialized.
```

Comment:
Should the "Example" section have real, live working code that one could copy and paste?  This is not an .md to .rst issue, but one relating to standardization.  This issue does not have to be dealt with in the conversion exercise.  I mention it because we probably want to move towards "live", working code, vs. pseudo code.